reuse/contribute-to-docs.txt

```diff
@@ -28,18 +28,101 @@ Use the :guilabel:`Give feedback` button at the top of any page to create a GitH
 New content
 -----------
 
-While contributing new content, it is easier to work with the docs on your local machine. You can submit a PR after all the checks have passed and things looks satisfactory. You'll need ``make`` and ``python3`` installed on your system.
+When adding new content, it's easier to work with the documentation on your local machine. Once you've made your changes, ensure all checks have passed and everything looks satisfactory before submitting a pull request (PR). To run the checks, you must have ``make`` and ``python3`` installed on your system.
+
```

Comment:
How about using a slightly different ordering of the subsections to match the expected workflow (& to indicate that certain steps are optional):
- Download and install the doocs
- Build and serve the docs
- Create content
	- Create new categories (optional)
	- Create new subcategories (optional)
	- Create new pages
- Perform checks and submit a PR